networks.json

```diff
@@ -0,0 +1,19 @@
+{
+  "GPv2AllowListAuthentication": {
+    "4": {
+      "address": "0x8D8C520c2073f7F4684A6c0c36FE05821173b972",
+      "transactionHash": "0xc22b40872f4cc483aa61e830c30429b995712c0a11a6416773c17a4d78a8f73f"
+    }
+  },
+  "GPv2Settlement": {
+    "4": {
+      "address": "0x91D6387ffbB74621625F39200d91a50386C9Ab15",
+      "transactionHash": "0xd39a37b72a0ba3f6e7d05575f98494d54b5b81700366b0cfdbb9215fc218c994"
+    }
+  },
+  "GPv2AllowanceManager": {
+    "4": {
+      "address": "0x90c25afb622BBf85a7b7d48820B554b4A13bc31F"
```

Comment:
nit: Shouldn't this have the same tx hash as GPv2Settlement?

Reply:
That's a good point. Originally, I had that but I thought it was a little weird as the transaction wasn't for deploying the allowance manager but instead the settlement contract.

But if the idea is to be able to find the "genesis" for the contract then it makes sense to include it. I will add it back in.

Reply:
Note, that the tx hash for the settlement contract is also a call (which internall invokes create) and not a vanilla contract creation.

Reply:
> Note, that the tx hash for the settlement contract is also a call

Oh yeah! I always forget the magic of deterministic dpeloyments...